WordPress/src/main/java/org/wordpress/android/ui/reader/services/ReaderCommentService.java

```diff
@@ -31,18 +30,29 @@
 
 public class ReaderCommentService extends Service {
 
-    private static final String ARG_POST_ID   = "post_id";
-    private static final String ARG_BLOG_ID   = "blog_id";
-    private static final String ARG_NEXT_PAGE = "next_page";
+    private static final String ARG_POST_ID     = "post_id";
+    private static final String ARG_BLOG_ID     = "blog_id";
+    private static final String ARG_COMMENT_ID  = "comment_id";
+    private static final String ARG_NEXT_PAGE   = "next_page";
 
-    public static void startService(Context context, ReaderPost post, boolean requestNextPage) {
+    private static int mCurrentPage;
+
+    public static void startService(Context context, long blogId, long postId, boolean requestNextPage) {
         Intent intent = new Intent(context, ReaderCommentService.class);
-        intent.putExtra(ARG_BLOG_ID, post.blogId);
-        intent.putExtra(ARG_POST_ID, post.postId);
+        intent.putExtra(ARG_BLOG_ID, blogId);
+        intent.putExtra(ARG_POST_ID, postId);
         intent.putExtra(ARG_NEXT_PAGE, requestNextPage);
         context.startService(intent);
     }
 
+    public static void startServiceForComment(Context context, long blogId, long postId, long commentId) {
```

Comment:
This could use a comment explaining that it will keep requesting comments until the passed `commentId` is retrieved.